CHANGELOG.md

```diff
@@ -5,7 +5,31 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
-## [0.5.0] - 2015-12-22
+
+## [0.5.1] - 2016-2-04
+### Changed
+- Vanagon now assumes anything that is not listed as an archive is just
+  a file, and thus NON-ARCHIVE-EXTENSIONS no longer needs to be updated
+  for new extentions.
+- Vanagon now searches for full windows name in place of only searching
+  for "win"
+
+### Fixed
+- Builds will halt if any git command fails
+- Hardware engine will release locks on failures
+- Removed deprecated windows functions: drive_root and
```

Comment:
Same with this. I think this fixed functionality that wasn't available in the earlier release